roles/kubernetes/node/templates/kubelet-container.j2

```diff
@@ -1,16 +1,32 @@
 #!/bin/bash
-{{ docker_bin_dir }}/docker run --privileged \
---net=host --pid=host --name=kubelet --restart=on-failure:5 \
--v /etc/cni:/etc/cni:ro \
--v /opt/cni:/opt/cni:ro \
--v {{kube_config_dir}}:{{kube_config_dir}} \
--v /sys:/sys \
--v /dev:/dev \
--v {{ docker_daemon_graph }}:/var/lib/docker \
--v /var/run:/var/run \
--v /var/lib/kubelet:/var/lib/kubelet \
---memory={{ kubelet_memory_limit|regex_replace('Mi', 'M') }} --cpu-shares={{ kubelet_cpu_limit|regex_replace('m', '')  }} \
-{{ hyperkube_image_repo }}:{{ hyperkube_image_tag}} \
-nsenter --target=1 --mount --wd=. -- \
-./hyperkube kubelet \
-$@
+{{ docker_bin_dir }}/docker run \
+  --net=host \
+  --pid=host \
+  --privileged \
+  --name=kubelet \
+  --restart=on-failure:5 \
+  --memory={{ kubelet_memory_limit|regex_replace('Mi', 'M') }} \
+  --cpu-shares={{ kubelet_cpu_limit|regex_replace('m', '')  }} \
+  -v /etc/cni:/etc/cni:ro \
+  -v /etc/ssl:/etc/ssl:ro \
+  -v /etc/pki/tls:/etc/pki/tls \
+  -v /etc/pki/ca-trust:/etc/pki/ca-trust \
+  {%- if ansible_os_family in ["CoreOS", "Container Linux by CoreOS"] -%}
+  -v /usr/share/ca-certificates:/usr/share/ca-certificates:ro \
+  {%- else -%}
+  -v /usr/share/ssl:/usr/share/ssl:ro \
+  -v /usr/ssl:/usr/ssl:ro \
+  -v /usr/lib/ssl:/usr/lib/ssl:ro \
+  -v /usr/local/openssl:/usr/local/openssl:ro \
+  -v /var/ssl:/var/ssl:ro \
+  -v /etc/openssl:/etc/openssl:ro \
+  {%- endif -%}
+  -v /opt/cni:/opt/cni:ro \
+  -v /sys:/sys:ro \
+  -v /var/lib/docker/:/var/lib/docker:rw \
+  -v /var/lib/kubelet:/var/lib/kubelet:shared \
+  -v /var/run:/var/run:rw \
```

Comment:
ah, you placed `--pid=host` so that should be fine. But still, if not a typo, please remove rw, it's a default